### PR TITLE
[docs] update Register a tax certificate 

### PR DESCRIPTION
**Pull request by kanwal-kaur**
## Description

I updated tax certificate upload instructions because the support form now has the ability for a customer to be able to upload the certificate directly into the ticket as opposed to the support team asking for the certificate after the ticket is open. 

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/billing/tax-certificate.md [hunk 1]
```diff
@@ -29,13 +29,16 @@ Palo Alto, CA 94306
 ## Register a tax certificate
 
 1. [Submit a Docker Support ticket](https://hub.docker.com/support/contact) to initiate the process to register a tax certificate.
-2. Enter the required information. Select **Billing** from the **Topic** drop-down.
-3. In the **Additional Information** field, list the Docker ID/namespace(s) of the accounts that you want to apply the tax exemption certificate to.  
+2. Enter the required information.
+3. Select **Billing** from the **Topic** drop-down.
+4. Select **Tax Information** from the **Subtopic** drop-down.
```
Comment (dvdksn) [inline content/billing/tax-certificate.md:34]: Can we add query parameters to the contact url to pre-select these fields?

Comment (dvdksn) [reply content/billing/tax-certificate.md:34]: Yeah something like that

Comment (kanwal-kaur) [reply content/billing/tax-certificate.md]: Done. Thank you.

File: content/billing/tax-certificate.md [hunk 2]
```diff
@@ -28,14 +28,15 @@ Palo Alto, CA 94306
...
+1. [Submit a Docker Support ticket](https://hub.docker.com/support/contact?topic=Billing&subtopic=Tax%20information) to initiate the process to register a tax certificate.
+2. Enter the required information.
 3. In the **Additional Information** field, list the Docker ID/namespace(s) of the accounts that you want to apply the tax exemption certificate to.  
 
    > **Tip**
    >
    > You can list multiple namespaces that share the same tax exemption certificate, if applicable.
    { .tip }
-4. Select **Submit**.
+4. Select **Browse Files** to upload the tax certificate from your system or simply drag and drop the certificate in the given space.
```
Comment (dvdksn) [inline content/billing/tax-certificate.md:39]:
```suggestion
4. Add the tax certificate from your system by dragging and dropping them onto the file area, or select the **Browse Files** button to open a file dialog.
```

File: content/billing/tax-certificate.md [hunk 3]
```diff
@@ -28,14 +28,15 @@ Palo Alto, CA 94306
...
+5. Select **Submit**.
 
-Docker's Support team will reach out for you to submit your certificate once a ticket is open. You'll receive a confirmation from Docker once your tax exemption status has processed and is applied to your account.
\ No newline at end of file
+Docker's Support team will reach out to you once a ticket is open and if Support team needs any additional information. You'll receive a confirmation from Docker once your tax exemption status has processed and is applied to your account.
```
Comment (dvdksn) [inline content/billing/tax-certificate.md:42]:
```suggestion
Docker's support team will reach out to you if any additional information is required. You'll receive an e-mail confirmation from Docker once your tax exemption status is applied to your account.
```

File: content/billing/tax-certificate.md (hunk 2 above)
Comment (kanwal-kaur) [reply content/billing/tax-certificate.md]: Done. Thank you.

File: content/billing/tax-certificate.md (hunk 3 above)
Comment (kanwal-kaur) [reply content/billing/tax-certificate.md]: Done. Thank you.

**Review by dvdksn — APPROVED** [review]

**Pull request merged**